docs/src/components/Layout/FrameworkChooser.tsx

```diff
@@ -26,7 +26,7 @@ const FrameworkLink = ({
     : `/${framework}`;
 
   return (
-    <Link href={isDisabled ? '#' : href} passHref>
```

Comment:
Do we know why this logic was here originally?

Reply:
Yes, it was added 1 month ago in this PR https://github.com/aws-amplify/amplify-ui/pull/2966/files#diff-b27004b57d34aded523c1d37576d69fd905d61aca4de4c19d7f3f0e8750a1857R29

Reply:
It was `<Link href={isDisabled ? '' : href} passHref>` to keep an empty `href` for disabled button. Since we don't want `href` to be empty, we don't have to differeciate disabled and enabled.

Reply:
Why did we have that conditional there to begin with?

Reply:
as mentioned in the description. The conditional would make all the buttons to be `react`

Reply:
Sure, but why did we want that behavior? Not that I'm against the change or anything, but the code change reads as an update to the UX of the site and I just want to understand the actual thoughts around the initial behavior

Reply:
Oh, there's no UX change. 2 reasons for this small change. 1) to make the links right, even though it's disabled. 2) to make sure the link-checker (links.cy.ts) passes

Reply:
I'm still confused about how the links will remain disabled? I think this change makes them always enabled because the disabled value on button will have no effect on the a tag.

Reply:
> I'm still confused about how the links will remain disabled? I think this change makes them always enabled because the disabled value on button will have no effect on the a tag. Can you please test that the links remain disabled and if you click on them nothing happens?

Good point...Those looks disabled links are clickable 😂

Reply:
OK! I made the link conditional. If it's disabled, there's no link around the button

Reply:
Rather than making a new component, you can just assign a `frameworkButton` variable and conditionally wrap it with the `Link`, example:
```
const FrameworkLink = ({
  framework,
  onClick,
  isDisabled,
}: FrameworkLinkProps) => {
  const { pathname, query } = useRouter();
  const isCurrent = query.platform === framework;
  const classNames = `docs-framework-link ${isCurrent ? 'current' : ''}`;
  const href = pathname.includes(platformPath)
    ? pathname.replace(platformPath, framework)
    : `/${framework}`;

  const frameworkButton = (
    <Button
      size="small"
      as="a"
      className={classNames}
      onClick={onClick}
      isDisabled={isDisabled}
    >
      <FrameworkLogo framework={framework} className="docs-framework-img" />
      {FRAMEWORK_DISPLAY_NAMES[framework]}
    </Button>
  );

  return isDisabled ? (
    frameworkButton
  ) : (
    <Link href={href} passHref>
      {frameworkButton}
    </Link>
  );
};
  ```

Reply:
Good point @calebpollman . updated!